pkg/ipam/allocator/multipool/allocator.go

```diff
@@ -86,11 +87,11 @@ func (a *Allocator) Init(ctx context.Context) (err error) {
 		}
 
 		log.WithFields(logrus.Fields{
-			"pool-name":     poolName,
-			"ipv4-cidrs":    pool.ipv4CIDRs,
-			"ipv4-masksize": pool.ipv4MaskSize,
-			"ipv6-cidrs":    pool.ipv6CIDRs,
-			"ipv6-masksize": pool.ipv6MaskSize,
+			"pool-name":      poolName,
+			"ipv4-cidrs":     pool.ipv4CIDRs,
+			"ipv4-mask-size": pool.ipv4MaskSize,
+			"ipv6-cidrs":     pool.ipv6CIDRs,
+			"ipv6-mask-size": pool.ipv6MaskSize,
 		}).Debug("adding pool")
 		if addErr := a.poolAlloc.AddPool(poolName, pool.ipv4CIDRs, pool.ipv4MaskSize, pool.ipv6CIDRs, pool.ipv6MaskSize); addErr != nil {
```

Comment:
Should we actually remove the `operatorOption.Config.IPAMMultiPoolMap` as part of this PR too?

Reply:
Good point. I've added a separate commit removing the option and related parsing code.